specification/paths/MultiColliShipment.json

```diff
@@ -0,0 +1,52 @@
+{
+  "post": {
+    "tags": [
+      "Multi-colli shipment"
+    ],
+    "summary": "Create a multi-colli shipment.",
+    "description": "This endpoint creates a new multi-colli shipment in the system of the carrier.",
+    "parameters": [
+      {
+        "in": "body",
+        "name": "Multi-colli shipment",
+        "description": "The multi-colli shipment to be created.",
+        "required": true,
+        "schema": {
+          "type": "object",
+          "required": [
+            "data"
+          ],
+          "additionalProperties": false,
+          "properties": {
+            "data": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Shipment"
+              }
+            }
```

Comment:
What about the `meta`?
I think we should support the same meta as the shipment endpoint (which will get applied to all shipments).

Reply:
I initially had it in there, but didn't know if it was a requirement. I can add it back.